nucliadb_texts/src/reader.rs

```diff
@@ -136,13 +177,13 @@ impl ReaderChild for TextReaderService {
         let mut keys = vec![];
         let searcher = self.reader.searcher();
         for addr in searcher.search(&AllQuery, &DocSetCollector)? {
-            let Some(key) = searcher
+            let key = searcher
                 .doc(addr)?
                 .get_first(self.schema.uuid)
-                .and_then(|i| i.as_text().map(String::from))
-            else {
-                continue;
-            };
+                .expect("documents must have a uuid.")
```

Comment:
If we are indexing data without an uuid I think is appropriate to fail as spectacularly as possible, it would need to be fixed ASAP because data is being corrupted.